web/src/routes/timetable/+layout.svelte

```diff
@@ -1 +1,10 @@
-<slot />
+<script>
+import Navbar from '../../lib/elements/ui/navbar/navbar.svelte';
```

Comment:
```suggestion
import Navbar from '$lib/elements/ui/navbar/navbar.svelte';
```
should also work I think

Reply:
fixed